app/src/test/java/com/github/sdp/mediato/api/themoviedb/TMDBApiTest.java

```diff
@@ -30,7 +28,7 @@ public class TMDBApiTest {
         @NonNull
         @Override
         public MockResponse dispatch(@NonNull RecordedRequest recordedRequest) {
-            switch (recordedRequest.getPath()){
+            switch (recordedRequest.getPath()) {
                 case "/3/search/movie?api_key=apiKey&query=searchTerm&language=en-US&page=1":
```

Comment:
This is a hard test. We need to have constants.

Reply:
I will change it

Reply:
So I do not think that it is a good idea since changing one of the value must make the test to fail.
If the API is updated, having hard-coded values here will force the programmer to carefully each URL and map it to the corresponding APITestString value.